odoo/addons/base/transpiler/codes/alias.js

```diff
@@ -0,0 +1 @@
+/** @odoo-module alias=tests.Alias **/
```

Comment:
definitely need to ask if this is a proper way to test...

Reply:
No, it's not.